src/cli.rs

```diff
@@ -527,6 +527,35 @@ pub fn build_cli() -> Command {
                         .help("Path to a Docker config.json-like path. Can be used to indicate registry authentication details")
                 )
         )
+        .subcommand(
+            Command::new("save")
+                .about("save policies in tar file")
+                .arg(
+                    Arg::new("policies")
+                        .num_args(1..)
+                        .required(true)
+                        .help("list of policies to save")
+                )
+                .arg(
+                    Arg::new("output")
+                    .long("output")
+                    .short('o')
+                    .required(true)
+                    .value_name("PATH")
+                    .help("path where the file will be stored")
```

Comment:
This isn't really a path, the user has to provide the name of the file to write to

Reply:
https://github.com/kubewarden/kwctl/pull/348/commits/b5037621fe1e1730ce73087090b321ecfa560486